first_hour_on_vwb/creating_a_data_collection.ipynb

```diff
@@ -667,23 +677,30 @@
    "metadata": {},
    "source": [
     "---\n",
-    "Copyright 2023 Verily Life Sciences LLC\n",
+    "Copyright 2024 Verily Life Sciences LLC\n",
```

Comment:
I'm not 100% sure, but I think one is 'supposed' to keep the original copyright and not update it yearly.  At least that is what I was told at one time.

Reply:
Per [the Open Source Stack  Exchange](https://screenshot.googleplex.com/8Pa2iJA5XEKn8KE), it's appropriate to update the copyright year to reflect when changes have been made. It would not be appropriate to update it, in other words, if we were not also updating the content, but since I am, it is appropriate.